app/src/test/java/com/example/plantsapp/data/repository/RoomTasksRepositoryTest.kt

```diff
@@ -0,0 +1,173 @@
+package com.example.plantsapp.data.repository
+
+import com.example.plantsapp.data.dao.StubPlantWithTasksDao
+import com.example.plantsapp.data.entity.TaskKeys
+import com.example.plantsapp.data.utils.createPlant
+import com.example.plantsapp.data.utils.createRoomPlant
+import com.example.plantsapp.data.utils.createRoomTask
+import com.example.plantsapp.domain.model.Plant
+import com.example.plantsapp.domain.model.Task
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.test.TestCoroutineScope
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import java.util.Calendar
+
+@ExperimentalCoroutinesApi
+class RoomTasksRepositoryTest {
+
+    private val creationDate = Calendar.getInstance().apply {
+        set(2021, 11, 10)
+    }.time
+    private val defaultPlants = listOf(
+        createRoomPlant(
+            name = "Greg",
+            creationDateMillis = creationDate.time
+        ),
+        createRoomPlant(
+            name = "Roger",
+            creationDateMillis = creationDate.time
+        )
+    )
+    private val defaultTasks = listOf(
+        createRoomTask(
+            taskKey = TaskKeys.WATERING_TASK,
+            plantName = "Greg",
+            frequency = 11
+        ),
+        createRoomTask(
+            taskKey = TaskKeys.SPRAYING_TASK,
+            plantName = "Greg",
+            frequency = 2
+        ),
+        createRoomTask(
+            taskKey = TaskKeys.SPRAYING_TASK,
+            plantName = "Roger",
+            frequency = 6
+        ),
+        createRoomTask(
+            taskKey = TaskKeys.LOOSENING_TASK,
+            plantName = "Roger",
+            frequency = 12
+        )
+    )
```

Comment:
How about minify these  tasks and other things to use only those params that are needed?